include/certifier_framework.h

```diff
@@ -145,9 +145,11 @@ namespace certifier {
 
     class cc_trust_data {
 
-    static const int max_symmetric_key_size_ = 128;
```

Comment:
I remember the reason for this change. Are there other solutions?

Reply:
@yelvmw - I don't think there are other solutions. I tried to redefine this limit using an enum-constant, but even that definition ran into issues.

Here is an example of a compilation failure I got with one of the alt solutions attempt:

```
../include/certifier_framework.h:149:50: error: ‘MAX_SYMMETRIC_KEY_SIZE’ was not declared in this scope
  149 |       static const int max_symmetric_key_size_ = MAX_SYMMETRIC_KEY_SIZE;
      |                                                  ^~~~~~~~~~~~~~~~~~~~~~
```

Here's another example,

Both are defined as private:

```
148       enum {MAX_SYMMETRIC_KEY_SIZE = 128};
149       // Python swig bindings need this to be public, to size other array decls
150       static const int max_symmetric_key_size_ = MAX_SYMMETRIC_KEY_SIZE;
```

Compilation fails like:

```
certifier_framework_wrap.cc: In function ‘PyObject* _wrap_cc_trust_data_symmetric_key_bytes__set(PyObject*, PyObject*)’:
certifier_framework_wrap.cc:7149:64: error: ‘certifier::framework::cc_trust_data::<unnamed enum> certifier::framework::cc_trust_data::MAX_SYMMETRIC_KEY_SIZE’ is private within this context
 7149 |       for (; ii < (size_t)certifier::framework::cc_trust_data::MAX_SYMMETRIC_KEY_SIZE; ++ii) *(byte *)&arg1->symmetric_key_bytes_[ii] = *((byte *)arg2 + ii);
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~
In file included from certifier_framework_wrap.cc:2824:
../include/certifier_framework.h:148:13: note: declared private here
  148 |       enum {MAX_SYMMETRIC_KEY_SIZE = 128};
      |             ^~~~~~~~~~~~~~~~~~~~~~
certifier_framework_wrap.cc: In function ‘PyObject* _wrap_cc_trust_data_service_symmetric_key__set(PyObject*, PyObject*)’:
certifier_framework_wrap.cc:7715:64: error: ‘certifier::framework::cc_trust_data::<unnamed enum> certifier::framework::cc_trust_data::MAX_SYMMETRIC_KEY_SIZE’ is private within this context
 7715 |       for (; ii < (size_t)certifier::framework::cc_trust_data::MAX_SYMMETRIC_KEY_SIZE; ++ii) *(byte *)&arg1->service_symmetric_key_[ii] = *((byte *)arg2 + ii);
      |                                                                ^~~~~~~~~~~~~~~~~~~~~~
```